django/db/migrations/operations/models.py

```diff
@@ -56,12 +56,11 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         # Sanity-check that there are no duplicated field names, bases, or
         # manager names
         _check_for_duplicates('fields', (name for name, _ in self.fields))
-        _check_for_duplicates(
-            'bases',
-            (base._meta.label_lower if isinstance(base, models.base.ModelBase) else base.lower()
-             for base in self.bases
-             if base is not models.Model)
-        )
+        _check_for_duplicates('bases', (
+            base._meta.label_lower if hasattr(base, '_meta') else
```

Comment:
what if two mixins have the same name?

hypothetically:  `facebook.models.UserMixin` and `twitter.models.UserMixin`